iceoryx_posh/include/iceoryx_posh/internal/popo/building_blocks/condition_variable_signaler.hpp

```diff
@@ -28,9 +28,9 @@ class ConditionVariableSignaler
     explicit ConditionVariableSignaler(cxx::not_null<ConditionVariableData* const> condVarDataPtr) noexcept;
     virtual ~ConditionVariableSignaler() noexcept = default;
     ConditionVariableSignaler(const ConditionVariableSignaler& rhs) = delete;
-    ConditionVariableSignaler(ConditionVariableSignaler&& rhs) noexcept = default;
+    ConditionVariableSignaler(ConditionVariableSignaler&& rhs) noexcept = delete;
```

Comment:
Since we do not need it. The ConditionVariableSignaler is part of the `ChunkQueue` which is stored in the shared memory and never moved we only work with the pointer.